metamorph/build.gradle

```diff
@@ -25,10 +25,10 @@ dependencies {
   implementation project(':metafacture-io')
   implementation project(':metafacture-mangling')
   implementation project(':metafacture-javaintegration')
-  implementation 'org.slf4j:slf4j-api:1.7.21'
-  testImplementation 'junit:junit:4.12'
-  testImplementation 'org.mockito:mockito-core:2.5.5'
-  testRuntimeOnly 'org.slf4j:slf4j-simple:1.7.21'
+  implementation "org.slf4j:slf4j-api:${versions.slf4j}"
+  testImplementation "org.slf4j:slf4j-simple:${versions.slf4j}"
```

Comment:
Why did you change this `slf4j-simple` dependency from `testRuntimeOnly` to `testImplementation`?

Reply:
Thx again - was absently. Fixed with [8ec34cc](https://github.com/metafacture/metafacture-core/pull/489/commits/8ec34cc2b413cdfa0b3315af8344f7ab8af85849).